src/datachain/lib/dc.py

```diff
@@ -2500,8 +2508,10 @@ def to_storage(
         placement: FileExportPlacement = "fullpath",
         use_cache: bool = True,
         link_type: Literal["copy", "symlink"] = "copy",
+        num_workers: Optional[int] = EXPORT_FILES_MAX_THREADS,
```

Comment:
Please change the name to something else since we use term `worker` for distributed compute. `threads` looks like a better fit. 

`EXPORT_FILES_MAX_THREADS` also suggests using threads, not workers.